.github/workflows/cd-pipeline.yml

```diff
@@ -116,4 +116,22 @@ jobs:
     with:
       environment: prod
       ref: ${{ needs.create-and-post-tag.outputs.newVersion }}
-      lambdaDeploy: true
\ No newline at end of file
+      lambdaDeploy: true
+
+  send-slack-notifications:
+    needs: [deploy-to-prod, create-and-post-tag]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
should we be using actions/checkout@v4, which is specified earlier in this workflow?

Reply:
Oh good callout! yeah i 'll make that change now.